cuda_bindings/cuda/bindings/nvvm.pxd

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+# SPDX-License-Identifier: LicenseRef-NVIDIA-SOFTWARE-LICENSE
+#
+# This code was automatically generated across versions from 12.0.1 to 12.8.0. Do not modify it directly.
+
+from libc.stdint cimport intptr_t, uint32_t
+
+from .cynvvm cimport *
+
+
+###############################################################################
+# Types
+###############################################################################
+
+ctypedef nvvmProgram Program
```

Comment:
not a big deal, but I am scratching my head staring at the internal config and wondering why this is not `_Program`...

Reply:
Do you want me to debug this?

Reply:
It is not a blocker to this PR, just wondering in case it's a bug in the codegen (unlikely)

Reply:
I spent a few minutes looking around, two observations:

* The code here is an expansion of `$type_decls` in the template. I didn't see anything obvious in the codegen template files that I could change to make an underscore appear here. I tried to leave `'prefix_to_strip'` in config_nvvm.py undefined, but that didn't change the result of the codegen. 

* https://github.com/NVIDIA/cuda-python/blob/8ed9c0368ec9604ba252a2e862b46881e0259b28/cuda_bindings/cuda/bindings/nvjitlink.pxd#L16 is similar: I see `Handle`there, no underscore:

```
ctypedef nvJitLinkHandle Handle
```

> just wondering in case it's a bug in the codegen

It looks like a codegen feature? :-)